src/workloads/query/PercentilesWindow.yml

```diff
@@ -0,0 +1,188 @@
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *maxPhases
+      PhaseConfig:
+        Threads: 1
+        Repeat: 1
+        Database: *db
+        CollectionCount: 1
+        DocumentCount: 100000
+        BatchSize: 1000
+        Document:
+          _id: {^Inc: {start: 0}}
+          # This is the field we will be computing percentiles on.
+          data: {^RandomDouble: {distribution: normal, mean: 50.0, sigma: 10.0}}
+          # We will use this field to sort our documents and create range windows.
+          date: {^RandomDate: {min: "2013-01-01", max: "2023-01-01"}}
+
+- ActorFromTemplate:
+    TemplateName: ComputePercentile
+    TemplateParameters:
+      Name: "percentile_small_window"
+      Documents: {documents: [-500, 500]}
+      OnlyActiveInPhase: 1
+
+- ActorFromTemplate:
+    TemplateName: ComputePercentile
+    TemplateParameters:
+      Name: "percentile_large_window"
+      Documents: { documents: [0, "unbounded"]}
+      OnlyActiveInPhase: 2
+
+- ActorFromTemplate:
+    TemplateName: ComputePercentile
+    TemplateParameters:
+      Name: "percentile_cumulative_t_digest"
+      Documents: { documents: ["unbounded", 0]}
+      OnlyActiveInPhase: 3
+
+# The next phases compute percentiles over range windows. We do not expect to compare the
+# runtime of range and document based windows, since the sizes of these windows can vary
+# greatly. However, we still want to cover all use cases of window functions, and did compare
+# $percentile runtime to $minN for all types of windows during development.
+- ActorFromTemplate:
+    TemplateName: ComputePercentile
+    TemplateParameters:
+      Name: "percentile_small_range_window"
+      # If the unit was larger than 'hour' we would have nearly all the documents in every window,
+      # which would not test having a smaller bounded window.
+      Documents: { range: [-500, 500], unit: "hour"}
+      OnlyActiveInPhase: 4
+
+- ActorFromTemplate:
+    TemplateName: ComputePercentile
+    TemplateParameters:
+      Name: "percentile_large_range_window"
+      Documents: { range: [0, "unbounded"], unit: "year"}
+      OnlyActiveInPhase: 5
+
+# Run the same test as above with a tighter 'unit', changing the frequency of adding/removing values.
+- ActorFromTemplate:
+    TemplateName: ComputePercentile
+    TemplateParameters:
+      Name: "percentile_large_range_window_day"
+      Documents: { range: [0, "unbounded"], unit: "day"}
+      OnlyActiveInPhase: 6
+
+- ActorFromTemplate:
+    TemplateName: ComputePercentile
+    TemplateParameters:
+      Name: "percentile_range_use_t_digest"
+      Documents: { range: ["unbounded", 0], unit: "year"}
+      OnlyActiveInPhase: 7
+
+# Run the same test as above with a tighter 'unit', changing the frequency of adding/removing values.
+- ActorFromTemplate:
+    TemplateName: ComputePercentile
+    TemplateParameters:
+      Name: "percentile_range_use_t_digest_day"
```

Comment:
nit: I'd name it `percentile_range_tdigest_day` because a) "use" doesn't add any information, b) I believe it would be more likely for people to search for "tdigest" rather than "t_digest"